bloggies_backend/src/models/email.ts

```diff
@@ -39,7 +40,8 @@ export default class Email {
         }
 
         try {
-            await sgMail.send(msg);
+            const emailRes = await sgMail.send(msg);
+            console.log(emailRes);
             console.log('Email sent')
         } catch(err) {
             throw new ExpressError(`Err: ${err}`, 400);
```

Comment:
Would we also need an endpoint to send emails for reminding users of their membership expiring if they haven't contributed to the Learning Circle in a week?